_persistence/exec/readme.md

```diff
@@ -5,8 +5,8 @@ title: Exec - Persistence
 type: persistence
 description: "This service allows you to execute commands in the underlying OS to persist item states. It could be used to, for example, call the original `rrdtool` CLI tool."
 source: https://github.com/openhab/openhab1-addons/blob/master/bundles/persistence/org.openhab.persistence.exec/README.md
-since: 1x
-install: manual
+since: 2x
+install: auto
```

Comment:
The bug is [here](https://github.com/openhab/openhab-docs/pull/357/files#diff-62405a701f8fabb964be57ae00f32cdeR73) where I'm trying to mate up feature names with package names.  Since there is the exec persistence service with no feature defined yet, the referenced code is finding the exec transformation service by mistake, and so saying the exec persistence service is 2.x and auto install.  Lacking time ATM to fix it. It seemed liveable to have the transformation services documented and later either fix this bug, or just add a feature for the exec persistence service.